gno.land/cmd/gnoland/testdata/event.txtar

```diff
@@ -0,0 +1,60 @@
+# start a new node
+gnoland start
+
+gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/demo/ee -gas-fee 1000000ugnot -gas-wanted 2500000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+stdout 'GAS WANTED: 2500000'
+stdout 'GAS USED: [0-9]+'
+stdout 'EVENTS: null'
+
+gnokey maketx call -pkgpath gno.land/r/demo/ee -func Sender -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+stdout 'GAS WANTED: 2000000'
+stdout 'GAS USED: [0-9]+'
+stdout 'EVENTS: \[?.*sender.*SubSender.*receiver.*SubReceiver.*\]'
```

Comment:
By the way, we require unit tests as well. The txtar formats are helpful. However, this PR lacks a complete event output demonstration. It's crucial to ensure consistency across the entire object. Please include a test.go file with one or more unit tests. These tests will help us evaluate both the high-level behavior and the generated formats effectively.

Reply:
To identify the caller using the framestack, include a unit test where contractA calls a callback from contractB, with the callback emitting. This is important due to the lack of typesafety in this emit implementation. Ensure the contract initializing the event cannot be falsified but can provide a callback for later execution.

Reply:
Here's a possible scenario to come up mind:

```go
package cbee

import (
    "std"
)

const (
	sender = "sender"
	receiver = "receiver"
)

type contractA struct{}

func (c *contractA) sender(cb func()) {
	subSender()
	cb()
}

func subSender() {
	std.Emit(sender, "k1", "v1", "k2", "v2")
}

type contractB struct{}

func (c *contractB) subReceiver() {
	std.Emit(receiver, "bar", "baz")
}

func Foo() {
    a := &contractA{}
	b := &contractB{}

	a.sender(func() {
		b.subReceiver()
	})
}
```

When the `Foo` function is executed, first the `sender` event is triggered and its event data is passed. Then, the `receiver` event is triggered and its event data is passed, and so on.

Running it through txtar (unit test have not written yet). I can see that the `sender` type event shows that an event occurred in the `subsender`, and the `receiver` type event shows that an event occurred in the `subreceiver`.

```plain
EVENTS: ["{\"type\":\"sender\",\"pkg_path\":\"gno.land/r/demo/cbee\",\"identifier\":\"subSender\",\"timestamp\":1713769028,\"attributes\":[{\"key\":\"k1\",\"value\":\"v1\"},{\"key\":\"k2\",\"value\":\"v2\"}]}","{\"type\":\"receiver\",\"pkg_path\":\"gno.land/r/demo/cbee\",\"identifier\":\"subReceiver\",\"timestamp\":1713769028,\"attributes\":[{\"key\":\"bar\",\"value\":\"baz\"}]}"]
```